### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/guide/markdown/card.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#524] [Announcements]
# https://theme-hope.vuejs.press/zh/guide/markdown/card.html

你可以在 Markdown 中添加卡片。

https://theme-hope.vuejs.press/zh/guide/markdown/card.html

**Comment by chenweigao** [comment]
card 真帅

**Comment by 1226085293** [comment]
功能有点不够，比如固定宽高，网格容器，还有最主要的是 desc 不支持\n或者回车换行....

**Reply by Mister-Hope** [reply]
你说的功能是可以自己在外侧加一个div容器自己通过css实现的，我理解你可能需要一个更舒服的卡片，但每个人对合理的预设期待是不同的。

**Comment by TeslaPeer** [comment]
能有hover动态就好了

**Reply by Mister-Hope** [reply]
如果你添加了 link，那么应该有一个合理的hover动态。如出现问题或其他建议请开 Issue
